_posts/2017-03-06-new-topology-system.md

```diff
@@ -0,0 +1,102 @@
+2. **Memory**. We don't store, for example, a resname for each atom, but instead store attributes at the level they make sense for.
+3. **Consistency**. Since attributes are stored in one place, we avoid cases where the topology is in an inconsistent state, e.g. two atoms in the same residue give a different resname.
+4. **No staleness**. Because e.g. `ResidueGroup`s are only an array of indices, not a list of `Residue` objects generated upon creation of the group, changes of resiude-level properties by another `ResidueGroup` are always reflected consistently by every other one. Data is not duplicated anywhere in this scheme, and is all contained in the `Topology` object.
+
+For further performance comparisons, check out this [notebook](http://nbviewer.jupyter.org/gist/dotsdl/0e0fbd409e3e102d0458).
+
+## External changes that may affect how you use MDAnalysis
+
+Previously, every object except ``Atom`` subclassed from ``AtomGroup``.
+This meant that calling `.positions` of would give you the positions of the ``Atom``s contained within that group.
+
+Previous class structure:
+```
+Atom
+
+AtomGroup  -> Residue
+           -> ResidueGroup -> Segment
+                           -> SegmentGroup
+```
+New class structure:
+```
+Group    -> AtomGroup
+         -> ResidueGroup
+         -> SegmentGroup
+
+Atom
+Residue
+Segment
+```
+
+Now each object only contains information pertaining to that particular object.
+A ``Residue`` object only yields information about the residue; to get to the atoms, use ``Residue.atoms``.
+
+### Why this was changed
+
+Previously everything inheriting from ``AtomGroup`` made it unclear at what level of topology a given method or attribute was working on.
+For example, does ``ResidueGroup.charges`` give the charge of the residues or the atoms?
+Also, it was unclear what size a given output would be (see [issue 411](https://github.com/MDAnalysis/mdanalysis/issues/411)).
+
+### How to work around this
+
+To access atom-level information from anything that isn't an ``AtomGroup``, use the `.atoms` level accessor.
+For example, changing all `.positions` calls on anything that isn't an `AtomGroup` to `.atoms.positions`.
+
+
+## Going forward: what does this mean for MDAnalysis as a project?
+
+A major benefit of the new topology system is that information about the topology of a ``Universe`` is now completely encapsulated in the ``Topology`` object.
+This not only makes development and maintenance easier, but also opens the door to some exciting new possibilities as simulation systems grow larger.
+A single ``Topology`` object can now be cleanly shared by multiple ``Universe`` instances, each with their own trajectory reader(s), making common operations such as fitting a trajectory to a reference structure or doing parallel analysis of many trajectories more feasible for large systems.
```

Comment:
"more feasible" is weird in a post like this: either it make it feasible or not (and if not, just strike the sentence)